app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,40 @@
+class InventoriesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_ability
+
+  def set_ability
+    @ability = Ability.new(current_user)
+  end
+
+  def new; end
```

Comment:
- Since you are not making use of this action, I suggest you remove this action and its corresponding routes, please fix this 🙏